### DISCUSSION TITLE
HA in docker su pi4 + HACS "problematico"

### DISCUSSION THREAD

**Discussion by riccionese** [indomus/forum#963] [HUB: Home Assistant]
Mentre il "vecchio" pi4 con HA gira bello tranquillo con tutta la mia configurazione "di produzione" ho rimediato un altro pi4 su cui faccio prove di vario genere.
In questo momento sto sperimentando la containerizzazione degli applicativi con docker. (poi vorrei passare  anche a kubernetes)
Ho già pihole + unbound che vanno che è una meraviglia, più altre prove "didattiche".
Ovviamente sto provando con HA, seguendo le guide del sito (che ho pure capito !!) per una installazione da zero e anche HA non ha problemi.
I problemi nascono con HACS che non riesco a rendere stabile.
Dopo un po' si pianta con questo messaggio:

![image](https://user-images.githubusercontent.com/62329292/138959243-f4cb5d0a-4bd6-4639-9e07-170336bea96c.png)

e nel log trovo:
```
homeassistant    | 2021-10-26 22:35:53 ERROR (MainThread) [custom_components.hacs] GitHub API ratelimited - 955 remaining
homeassistant    | 2021-10-26 22:35:53 ERROR (MainThread) [custom_components.hacs] HACS is disabled - HacsDisabledReason.RATE_LIMIT
```
Ho provato sia con lo stesso  token che uso "in produzione" sia generandone uno nuovo ma non ci cavo gli zampetti
ed ho già rimosso e reinstallato HACS in tutte le modalità che ho trovato in rete
Suggerimenti ?

**Comment by snakuzzo** [comment]
https://community.home-assistant.io/t/github-rate-limit-error-hacs/229709/145

**Reply by riccionese** [reply]
stasera provo
tnx

**Reply by riccionese** [reply]
... e ovviamente funzia!
